test/functional/commands/find/by-accessibility-id-e2e-specs.js

```diff
@@ -13,17 +13,17 @@ describe('Find - accessibility ID', function () {
     driver = await initDriver(APIDEMOS_CAPS);
   });
   after(async () => {
-    await driver.deleteSession();
+    await driver.quit();
   });
   it('should find an element by name', async () => {
-    await driver.findElOrEls('accessibility id', 'Animation', false).should.eventually.exist;
+    await driver.elementsByAccessibilityId('Animation').should.eventually.exist;
```

Comment:
Should be `elementByAccessibilityId`

Reply:
It should work either way.

Reply:
Is it any chance that `driver.elementsByAccessibilityId('Animation').should.eventually.exist` can throws exception?

Reply:
Yeah good point. An empty array wouldn't throw an exception.